icicle_v3/tests/test_field_api.cpp

```diff
@@ -224,31 +260,34 @@ TYPED_TEST(FieldApiTest, ntt)
     init_domain_config.is_async = false;
     init_domain_config.ext.set(CUDA_NTT_FAST_TWIDDLES_MODE, true);
 
-    ICICLE_CHECK(ntt_init_domain(scalar_t::omega(logn), init_domain_config));
-
     auto config = default_ntt_config<scalar_t>();
+    // config.stream = stream;
+    // config.coset_gen = coset_gen; // TODO - Implement. default: S::one()
+    config.batch_size = batch_size;       // default: 1
+    config.columns_batch = columns_batch; // default: false
+    config.ordering = ordering;           // default: kNN
+    config.are_inputs_on_device = true;   // TODO, ask yuval why set to true?
```

Comment:
for CPU it doesn't matter but for devices with own memory, like GPU/ZPU/FPGA, we need to know where the data resides.

Reply:
ok